pkg/rancher-desktop/hocs/withCredentials.ts

```diff
@@ -0,0 +1,40 @@
+import Vue, { VueConstructor, VNode } from 'vue';
```

Comment:
I can't figure out what `hocs` is supposed to stand for. Could you replace with a more self-explanatory name?

Reply:
HOCs = Higher Order Components. I would place this under `composables` if we were using the composition api/Vue 3.

I'm open to a suggestions.

Reply:
ok, sounds fine.